test/cql-pytest/test_describe.py

```diff
@@ -447,6 +447,49 @@ def test_desc_index(cql, test_keyspace):
         
         assert f"CREATE INDEX named_index ON {tbl}{maybe_space}(c)" in c_desc
 
+def test_desc_index_on_collections(cql, test_keyspace):
```

Comment:
Does this test pass on Cassandra as well? If not, please think if it shouldn't, and consider adding the appropriate tag (xfail/scylla_only/cassandra_bug) depending on why it works differently in Cassandra.

Reply:
Yes, it passes but it required a small change in assertions (see comment in the code).

In short: Scylla outputs `CREATE INDEX ON table_name(column_name)`  but Cassandra does `CREATE INDEX ON table_name (column_name)` (extra whitespace if you didn't noticed :D )